.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Bug Report
+about: Report a bug/issue that you found in the SVGs, CSS files, etc...
```

Comment:
```suggestion
about: Report a bug/issue that you found in the SVGs, CSS files, etc..
```

Reply:
I was curious about and search it up. Turns out we only need 1 period after etc.